addons/bus/controllers/main.py

```diff
@@ -38,3 +38,7 @@ def poll(self, channels, last, options=None):
         if request.registry.in_test_mode():
             raise exceptions.UserError(_("bus.Bus not available in test mode"))
         return self._poll(request.db, channels, last, options)
+
+    @route('/longpolling/im_status', type="json")
```

Comment:
can you make it explicit that this is `auth="user"`? (not even sure that auth=user is really the default)